src/work-allocation-2/models/cases/invoked-case-action.model.ts

```diff
@@ -0,0 +1,6 @@
+import { Case, CaseAction } from '.';
+
+export default interface InvokedCaseAction {
+  case: Case;
```

Comment:
_Question:_
Is using `case` safe here? Asking for a friend. 😁

Reply:
It worked but it wasn't safe so I've renamed it (and all references to it) to invokedCase